src/model_theory/terms_and_formulas.lean

```diff
@@ -362,6 +407,66 @@ begin
       { exact (dif_neg (λ h', h (nat.lt_of_add_lt_add_left h'))).trans (dif_neg h).symm } } }
 end
 
+lemma realize_lift_at {n n' m : ℕ} {φ : L.bounded_formula α n}
```

Comment:
I have to run now, but it's interesting how this is formulated differently from `realize_formula_subst_lift` in Flypitch.